src/site/content/en/lighthouse-performance/efficient-animated-content/index.md

```diff
@@ -64,6 +65,19 @@ Luckily, you can recreate these behaviors using the `<video>` element.
 </video>  
 ```
 
+## Use a service that converts GIFs to HTML5 videos
+
+Many [image CDNs](/image-cdns/) support GIF to HTML5 video conversion. You upload a
+GIF to the image CDN, and the image CDN returns an HTML5 video.
+
```

Comment:
@paulirish in #4093 you had this listed as stack-specific guidance for Drupal, Joomla, and WordPress. The fact that the text was the same in each section suggests that it's not stack-specific guidance (by definition) so I moved it up into its own section.

https://github.com/GoogleChrome/web.dev/pull/4093/files#diff-7e25d666eee78bb880715d980b8cee454af0c39a7505421d3bc77e6828cc3614R75